### PR TITLE
[SPARK-15322][mllib][core][sql]update deprecate accumulator usage into accumulatorV2 in spark project

### PR DESCRIPTION
**Pull request by WeichenXu123**
## What changes were proposed in this pull request?

I use Intellj-IDEA to search usage of deprecate SparkContext.accumulator in the whole spark project, and update the code.(except those test code for accumulator method itself)
## How was this patch tested?

Exisiting unit tests

### CODE REVIEW

**Comment by yanboliang** [comment]
This looks good. ping @mengxr @jkbradley @MLnick Could you help to add @WeichenXu123 to whitelist? Thanks.

**Comment by MLnick** [comment]
Jenkins add to whitelist

File: mllib/src/main/scala/org/apache/spark/ml/util/stopwatches.scala [hunk 1]
```diff
@@ -19,7 +19,8 @@ package org.apache.spark.ml.util
 
 import scala.collection.mutable
 
-import org.apache.spark.{Accumulator, SparkContext}
+import org.apache.spark.{SparkContext}
+import org.apache.spark.util.LongAccumulator;
```
Comment (HyukjinKwon) [inline mllib/src/main/scala/org/apache/spark/ml/util/stopwatches.scala]: (The imports might have to be cleaned up as below:)

```scala
import org.apache.spark.SparkContext
import org.apache.spark.util.LongAccumulator
```

**Comment by srowen** [comment]
Rather than change this in just a couple places, can you update all internal usages of the old accumulator API?

**Comment by WeichenXu123** [comment]
@srowen I use Intellj-IDEA to search usage of deprecate SparkContext.accumulator in the whole spark project, and   update the code.(except those test code for accumulator method itself)
@HyukjinKwon I update  import org.apache.spark.{SparkContext} ==> import org.apache.spark.SparkContext

**Comment by srowen** [comment]
Those do look like instances that can be changed, yes. LGTM

**Comment by WeichenXu123** [comment]
@srowen updated. Seems no problem.

**Comment by srowen** [comment]
Merged to master/2.0

**Pull request closed without merging**